baseplate/experiments/providers/simple_experiment.py

```diff
@@ -27,6 +27,55 @@
 }
 
 
+def _generate_overrides(override_config):
+    """Generate a dictionary of overrides.
+
+    The format of the override config is expected to be a list of dicts,
+    where each dict contains a single key (the treatment to assign) and value
+    (the targeting tree for that treatment).
+
+    This combination is used to ensure that the ordering of the overrides is
+    maintained, allowing consistent overriding for any overlapping area
+    across targeting trees. Each incoming variant request will check each
+    targeting override in sequence until one is matched or none match.
+
+    If the override configuration does not contain a wrapping list (ie, it
+    is just a dictionary), then overrides will work, but override ordering
```

Comment:
Why even give the option of this?  Couldn't we just say "you have to give a list of dicts"?

Reply:
We could; I wanted to give the option of simpler configs if you're not going to have any overriding sets (ie, overriding specific users, for instance). I'm happy to remove it, but I generally like a bit more flexibility when it doesn't greatly increase complexity.

Reply:
I imagine most if not all of these configs are going to be generated by code rather than handwritten so the little bit of configuration overhead probably isn't a big deal.  While the code is straight forward, I feel like having forked behavior like this can lead to unexpected results (not bugs, just someone makes a config that doesn't do what they expect) and confusion about what exactly is the "right" thin to do.

Reply:
fair point. Fixed

Reply:
I think this docstring still needs to be updated to reflect that change.

Reply:
updated